manifests/reposync-crd.yaml

```diff
@@ -1280,6 +1280,38 @@ spec:
                       about valid inputs: https://pkg.go.dev/time#ParseDuration. Recommended
                       apiServerTimeout range is from "3s" to "1m".'
                     type: string
+                  containerOverrides:
```

Comment:
wait wait wait... we can't call this containerOverrides if it doesn't include the container resource overrides...

Reply:
@sdowell it looks you maybe you requested this?

Reply:
I [offered this as a suggestion but acknowledged that this name probably isn't feasible with the presence of the other resource overrides field](https://github.com/GoogleContainerTools/kpt-config-sync/pull/677#discussion_r1275211852). Looks like my other comment got resolved. @victorpras might help to let the OP resolve comments

Reply:
These are already under `override`. So -Override is redundant in a sub-field... and we already have override.resources and other container-specific overrides that we can't change now due to needing reverse compat.

I think logLevels was fine... I agree having logLevel in each entry is redundant, that's why I suggested just "level" but I';d rather have `spec.override.logLevels[].logLevel` than `spec.override.containerOverrides[].logLevel`... we're not designing this from scratch.

Reply:
reverting back to `logLevels`